doc/nrf/libraries/bluetooth_services/services/nus.rst

```diff
@@ -10,6 +10,7 @@ Nordic UART Service (NUS)
 The Bluetooth® LE GATT Nordic UART Service is a custom service that receives and writes data and serves as a bridge to the UART interface.
 
 The NUS Service is used in the :ref:`peripheral_uart` sample.
+It is also included with the Thingy:91 :ref:`connectivity_bridge`, but is disabled by default.
```

Comment:
Do we want to mention here all this service use cases?
There are more than these two

Reply:
I'm neutral on this, but it's worth noting that``:ref:`peripheral_uart```  and  ``:ref:`connectivity_bridge``` are arguably the two  most relevant use cases since the former is a direct demonstration, and the latter is essentially the de facto way to do wireless monitoring on Thingy:91

Reply:
Your call @KAGA164.

Reply:
We can keep it as it is now. I would like to avoid having all use cases under services with one exception, if this would be possible to be generated automatically but it is something that might come in future